hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/client/RemoteHTable.java

```diff
@@ -77,6 +73,10 @@
 import org.slf4j.LoggerFactory;
 
 import org.apache.hbase.thirdparty.com.google.common.base.Preconditions;
+import org.apache.hbase.thirdparty.com.google.protobuf.Descriptors;
+import org.apache.hbase.thirdparty.com.google.protobuf.Message;
+import org.apache.hbase.thirdparty.com.google.protobuf.Service;
+import org.apache.hbase.thirdparty.com.google.protobuf.ServiceException;
 
 /**
  * HTable interface to remote tables accessed via REST gateway
```

Comment:
This class should be removed?

Reply:
No? It is the Table implementation for rest.